### PR TITLE
chore(RHTAPWATCH-348): Add SOP URL to alert rule PodsNotReady

### PR DESCRIPTION
**Pull request by Kousalya1998**
- Add SOP URL to PodsNotReady alert rule and its test case file

### CODE REVIEW

**Review by amisstea — APPROVED** [review]

File: test/promql/tests/test_notready_pods.yaml [hunk 1]
```diff
@@ -32,8 +32,12 @@ tests:
               phase: Pending
               source_cluster: cluster01
             exp_annotations:
-              summary: "Pod pod-1 is not ready"
-              description: "Pod pod-1 in namespace ns-1 on cluster cluster01 is not ready for more than 15 minutes."
+              summary: >-
+                Pod pod-1 is not ready
+              description: >-
+                Pod pod-1 in namespace ns-1 on cluster cluster01 is
```
Comment (hmariset) [inline test/promql/tests/test_notready_pods.yaml:38]: @Kousalya1998 yaml-lint is failing at this line with trailing space error. Please remove extra space at the end. Everything else looks good.

**Review by hmariset — CHANGES_REQUESTED** [review]

File: test/promql/tests/test_notready_pods.yaml (hunk 1 above)
Comment (Kousalya1998) [reply test/promql/tests/test_notready_pods.yaml:38]: I updated. Guess /retest was required to update that page.

**Review by hmariset — APPROVED** [review]
lgtm

**Pull request merged**